package.json

```diff
@@ -37,7 +37,7 @@
     "download:kubectl": "yarn run ts-node build/download_kubectl.ts",
     "download:helm": "yarn run ts-node build/download_helm.ts",
     "build:tray-icons": "yarn run ts-node build/build_tray_icon.ts",
-    "lint": "yarn run eslint --ext js,ts,tsx --max-warnings=0 .",
+    "lint": "NODE_ENV=production yarn run eslint --ext js,ts,tsx --max-warnings=0 .",
```

Comment:
This seems like an abuse of a `NODE_` prefixed environment variable

Reply:
Replaced to just `ENV`